gnome-initial-setup/pages/keyboard/cc-input-chooser.c

```diff
@@ -169,7 +169,14 @@ preview_cb (GtkLabel       *label,
 	InputWidget *widget;
 	const gchar *layout;
 	const gchar *variant;
-	gchar *commandline;
+	gchar *lc_msg = setlocale (LC_MESSAGES, NULL);
+	gchar *command[] = {"/usr/bin/gkbd-keyboard-display", "-l", NULL, NULL};
```

Comment:
Instead of packing allocated memory into `command[2]`, I'd either make this a const array and then still use the `commandline` variable as before, together with something like `g_shell_parse_argv()`, or make it a `GPtrArray` and add to it